feature-utils/poly-import/utils/status.js

```diff
@@ -1,23 +1,7 @@
-export function createSuccessStatus() {
-    return {
-        name: "SUCCESS",
-        isSuccess: true,
-    };
-}
-
-export function createWarningStatus(message) {
-    return {
-        name: "WARNING",
-        message,
-        isWarning: true,
-    };
-}
-
-export function createErrorStatus(error) {
-    return {
-        name: "ERROR",
-        error,
-        message: error.name,
-        isError: true,
-    };
+export class Status {
+    constructor({ name, isSuccess, message = null }) {
+        this.name = name;
+        this.isSuccess = isSuccess;
```

Comment:
Would it be possible to make this a computed attribute? It's always success when it is, not success when it's not, right?

Reply:
that's a nice idea :D

Reply:
then maybe the name should be an enum to avoid putting in the wrong string?